master/docs/developer/cls-buildsteps.rst

```diff
@@ -584,7 +584,7 @@ This class can only be used in new-style steps.
         Determine if the given path exists on the worker (in any form - file, directory, or otherwise).
         This uses the ``stat`` command.
 
-    .. py:method:: runGlob(glob)
```

Comment:
I don't know why I wrote runGlob(test). I mean runGlob(path), as the following documents the param 'path'